lib/DescriptionFileUtils.js

```diff
@@ -189,6 +189,8 @@ function getField(content, field) {
  */
 function cdUp(directory) {
 	if (directory === "/") return null;
+	if (directory.match(/^[a-zA-Z]:$/)) return null;
```

Comment:
Why we need it? We have unix path internally...

Reply:
Because it can be win32 path still. Of course, it has posix char separator, but absolute path can be resolved as `X:/something/` for example and for this case - to prevent this function to go up